### DISCUSSION TITLE
Better CSS syntax highlighting in custom CSS editor and UI element documentation

### DISCUSSION THREAD

**Discussion by Skyweaver-real** [stoatchat/discussions#536] [Feature Suggestions]
I've been customizing Revolt with custom css, but when I want to style a certain element I always have to use inspect element to find the element name, only for it to be some gibberish (ex: eiGSNL for the image viewer. ). Could there be some document explaining the UI elements' css and class name? Also, syntax highlighting would help.

**Comment by carolgram** [comment]
They don't want to make it easy to use their source. 😂 I doubt they even know, hence why they are rebuilding it all.  Hopefully the next version will not be such a mess. Having to Google how to invite people to your server and still struggling just goes to show that they got a long way to go
